sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/ContentSourceKind.java

```diff
@@ -11,6 +11,12 @@
 /** Type of content source. */
 public final class ContentSourceKind extends ExpandableStringEnum<ContentSourceKind> {
 
+    /**
+     * Creates or finds a ContentSourceKind from its string representation.
+     */
+    public ContentSourceKind() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.